### ISSUE TITLE
Banned from the discord server for being muslim

### ISSUE THREAD

**Issue by Mistik** [AssemblyScript/assemblyscript#1594]
Hi

Not really an issue with the code, but I was on the AssemblyScript discord in voice chat, and some trolls joined and insulted me, and islam, I then mentioned I was muslim and then got banned for some reason.

**Comment by dcodeIO** [comment]
This is what I recall:

* A working group meeting was taking place and you joined, asking questions that other participants kindly answered
* Suddenly, various people new to the server joined the meeting voice channel, making all sorts of inappropriate comments
* The disruption lasted for a few minutes with more people joining before we were able to figure out what happened:
  * A direct invite link to the meeting's voice channel has been created from your account according to the server logs
  * Those disrupting the meeting apparently followed the invite link, most likely on foreign channels
* As a mitigation, I took the following measures so we were able to continue with the meeting:
  * I deleted the invite link and disallowed the creation of new direct invite links to the meeting voice channel
  * I temporarily restricted access to the server for those disrupting the meeting, including your account as the creator of the invite link
* When the meeting was over about half an hour later, I immediately lifted all access restrictions again, so everyone with a legitimate interest in participating in the AssemblyScript community would be able to join again

I'd like to assure you that we strive to make AssemblyScript an open and welcoming environment for everyone, regardless of religion, and from what I remember no religion was mentioned during the disruption. Other participants may be able to confirm. If you however still do believe that you have been treated in violation of our [Code of Conduct](https://github.com/AssemblyScript/assemblyscript/blob/master/CODE_OF_CONDUCT.md), and since I was the person applying the access restrictions, I recommend to follow up with either @MaxGraey or @torch2424 with your complaint.

**Comment by torch2424** [comment]
Hello!

@dcodeIO Your recollection is 100% correct. :smile: Thankfully Discord keeps a log of who created invites, and how many uses there were, and we were able to identify that it was created by @Mistik . Then during the meeting, users rapidly joined making inappropriate comments (Not about religion, mostly things like "Woah, WTF is this?") during the meeting (and kind of ruined it). After looking more into it, it seems to be a common "trolling" practice called [Discord Raiding](https://www.reddit.com/r/discordapp/comments/bxjowu/why_do_people_raid_discord_servers/). 

I would also like to echo: 

> I'd like to assure you that we strive to make AssemblyScript an open and welcoming environment for everyone, regardless of religion, and from what I remember no religion was mentioned during the disruption. Other participants may be able to confirm.

I can confirm this as well (as the host of the meeting). We have a code of conduct, and we strive to uphold it. If you have information on who these trolls were, I will happily take the precautions to ensure that it does not happen again on any medium in which the AssemblyScript community is expected to interact :smile: 

---

With that, I will say I do think your game that we had chatted about that uses AssemblyScript is super interesting. I'd be happy to help with any AssemblyScript issues you have while building your project. 

I'm going to close this issue for now, but again, feel free to continue the discussion here.

**Comment by DaAwesomeRazor** [comment]
I was there and was sent a invite by the user “Henopied” asking to raid the server. I do recall in the VC Henopied was insulting islam and then Mistik got banned. It looked like Henopied was invited by Misitk but then used that trust and abused it by trolling and using Mistiks invite to bring in more trolls. When I joined I realized you guys were just chilling so I just sat there and listened to what you guys were talking about and discussed a bit about the library. Sad that trolls just try to ruin everything.
